processes/documents/Managing Operation Decision Making Meeting --fa.md

```
@@ -73,58 +73,59 @@
-فرآیند مدیریت جلسات تصمیم‌گیری عملیاتی شامل سه بخش **قبل از جلسه**، **در طول جلسه** و **پس از جلسه** است. مدیر جلسه در هر یک از این بخش‌ها وظایف مشخصی دارد که در ادامه به تفصیل شرح داده می‌شود.
+فرآیند مدیریت جلسات تصمیم‌گیری عملیاتی شامل سه بخش قبل از جلسه، در طول جلسه و پس از جلسه است. مدیر جلسه در هر یک از این بخش‌ها وظایف مشخصی دارد که در ادامه به تفصیل شرح داده می‌شود.
 
 
 #### قبل از جلسه
 
 مدیر جلسه عملیاتی باید پیش از شروع جلسه موارد زیر را انجام دهد:
 
-۱- **بررسی نحوه حضور منتورها در جلسات:**
-   - تمامی منتورها باید در اسپرینتی که پس از فرآیند آنبوردینگ انتخاب کرده‌اند، شرکت کنند.
+۱- بررسی نحوه حضور منتورها و همیارهای فنی در جلسات:
+   - منتورها و همیارهای فنی باید در تمامی جلسات اسپرینت، پس از آنبورد شدن و دریافت مسئولیتی در برنامه، حاضر شوند.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)